### ISSUE TITLE
make container logs in lagoon namespace configurable

### ISSUE THREAD

**Issue by tobybellwood** [uselagoon/lagoon-charts#405]
Some lagoon services should have log collection enabled to facilitate troubleshooting, whereas others won't need them (or are excessively chatty).

Perhaps adding a label to running pods to control collect/no-collect?

**Comment by smlx** [comment]
Yes, this whole functionality needs some reworking. Related: https://github.com/uselagoon/lagoon-charts/issues/321

**Comment by smlx** [comment]
The lagoon-logging chart doesn't disable logs collection from any `lagoon-*` namespace by default. Which namespaces specifically are you missing logs collection from?
